sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -115,20 +115,76 @@ await using (var client = new EventHubClient(connectionString, eventHubName))
     
     await using (EventHubConsumer consumer = client.CreateConsumer(consumerGroup, firstPartition, startingPosition))
     {
-        int maximumEventBatchSize = 25;
-        IEnumerable<EventData> eventBatch = await consumer.Receive(maximumEventBatchSize);
- 
-        // At this point, the eventBatch may have no events or may have as many as the maximum size requested, 
-        // depending on how many events were available in the partition.
+        using CancellationTokenSource cancellationSource = new CancellationTokenSource();
+        cancellationSource.CancelAfter(TimeSpan.FromSeconds(45));
+    
+        await foreach (EventData receivedEvent in consumer.SubcribeToEvents(cancellationSource.Token))
+        {
+            // At this point, the loop will wait for events to be available in the partition.  When an event 
+            // is available, the loop will iterate with the received event available.  Because we did not 
+            // specify a maximum wait time, the loop will wait forever unless cancellation is requested using
+            // the cancellation token.
+        }
     }
 }
 ```
 
+### Consume events from all partitions of an Event Hub
```

Comment:
Please consider this a basic skeleton;  @kinelski will revisit in the near future and smooth things out as well as correct any mistakes in the usage that I've made.

Reply:
How about `Consume events using an Event Processor` instead?

As a user, if I wanted to receive from all partitions, I could still very well use the previous sample and have it run for all partitions.

By having the words `Event Processor` in the header, I hope to draw attention to another way of consuming events that might be of interest to the user

Reply:
Sure.  worked into the samples PR.